libs/core/src/components/sage-button/stories/sage-button.stories.mdx

```diff
@@ -43,80 +50,50 @@ When following our usage guidelines, sage-button should be accessible right out
 
 Default button styles, used with affirmative actions such as submitting, continuing, and saving
 
-<Canvas>
-  <Story name="Primary" args={{ icon: '', disabled: false, slot: 'Default', type: 'button' }}>
-    {Template.bind()}
-  </Story>
-  <Story name="PrimaryDisabled" args={{ icon: '', disabled: true, slot: 'Disabled', type: 'button' }}>
-    {Template.bind()}
-  </Story>
-  <Story name="PrimaryIcon" args={{ icon: 'trashIcon', disabled: false, slot: 'Icon', type: 'submit' }}>
-    {Template.bind()}
-  </Story>
+<Canvas withSource="none">
+  <sage-button>Default</sage-button>
+  <sage-button disabled="true">Disabled</sage-button>
+  <sage-button icon="trashIcon">Icon</sage-button>
```

Comment:
Maybe I wasn't clear previously, but I was ok with multiple stories, e.g Primary, Secondary, Accent. I felt the slight variants `PrimaryIcon`, and `PrimaryDisabled` was a little too much.

Reply:
Gotcha, updates pushed 👍